examples/helloworld/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Files and directories created by pub
```

Comment:
I don't see `.gitignore` in other directories under `examples/`. We can remove this, right @johnpryan?

Reply:
Yup I realized it as well. There is already a parent `.gitignore`